src/main/resources/k8s/def/script.delete.all.sh

```diff
@@ -0,0 +1,23 @@
+echo "Deleting all application=windup"
+kubectl delete all -l application=windup -A
+echo "Deleting all application=windup-jon"
+kubectl delete all -l application=windup-jon -A
```

Comment:
I can see this label `windup-jon` multiple times but not so sure what it refers to.
Is this for testing purpose?

Reply:
It's the name of the examples/windup.yaml , but recently that file was changed inside to use "mta-example" so replaced here that name.